newsletter-2021-02.html

```diff
@@ -0,0 +1,92 @@
+<h1>tedana February 2021 Update</h1>
+Since our late December newsletter, we have created a new stable release available on PyPI!
+You can upgrade your local installation to the latest 0.0.9 version by running <xmp>pip install -U tedana</xmp>.
+We encourage everyone to go and check it out, as it includes many new bug fixes, improvements, and breaking changes.
+
+<h2>New Contributors</h2>
+We are excited to welcome three new contributors since our last newsletter: M. Versluis, Z. Ahmed, and A. Khojandi!
+Thanks to all of our contributors for their continued input and help in creating tedana.
+The community has now grown to 20 contributors, and we always look forward to seeing more new faces!
+If you’re not sure where to start, please feel free to open an issue or join our next
+<a href="https://tedana.readthedocs.io/en/latest/developing.html?highlight=developer%20call#monthly-developer-calls">monthly developer's call</a>.
+
+<h2>Governance Updates</h2>
+We have recently overhauled our governance structure, moving from a Benevolent-Dictator-for-Life (BDFL) model to a Steering committee structure.
+We have also implemented specific maintainer roles to better track project leadership, and we encourage contributors to self-describe their work on tedana in the new
+<a href="https://github.com/ME-ICA/tedana/blob/main/contributions.md">contributions description file</a>.
+You can learn more about our new governance structure and current project leadership in
+<a href="https://tedana.readthedocs.io/en/latest/governance.html">our documentation</a>.
+We encourage any community members who are interested in taking a more active role in tedana to open an issue and add themselves as maintainers!
+
+<h2>Code Enhancements</h2>
+<ul>
+    <li>New interactive reports!
+        Check them out!
+        User-facing documentation is available
+        <a href="https://tedana.readthedocs.io/en/latest/reporting.html">here</a>.
+        You can also see two example reports
+        <a href="https://me-ica.github.io/tedana-ohbm-2020/">here</a>.
+        </li>
+    <li>If no BOLD-like ICA components are found, tedana will now re-run the ICA step with a new seed.</li>
+    <li>Documentation has been improved both on readthedocs and inside of docstrings.</li>
+    <li>Added new documentation for multi-echo acquisition on Philips scanners <a href="https://tedana.readthedocs.io/en/latest/acquisition.html#philips">here</a>.</li>
+    <li>T2* values now have a nonzero floor.</li>
```

Comment:
```suggestion
    <li>T2* values now have a nonzero floor to prevent divide-by-zero errors during optimal combination.</li>
```

I'd move this to Bug Fixes TBH.